projects/media-viewer/src/lib/shared/viewer-operations.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * @deprecated Please DO NOT ADD to this file
+ * Instead please add new Operations to viewer.events.service.ts
+ */
```

Comment:
maybe we could keep these operations defined here and use them within the service

Reply:
Potentially, but they remain where they are for now. That is certainly up for discussion, hence the deprecation.